model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -107,6 +107,10 @@ def nodes_builder(model: GraphModule,
         else:
             raise Exception(f'Unknown node type: {node.name}')
 
+        # Disable activation
+        if node.target in [UNBIND,getattr]:
```

Comment:
Please don't merge it @ofirgo @lior-dikstein  this looks like a patch

Reply:
Please review again. If still it is not Ok let's talk.